src/Tools/dotnet-monitor/CollectionRules/Options/CollectionRuleDefaultsPostConfigure/EgressPostConfigure.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Actions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.CollectionRuleDefaultsInterfaces
+{
+    internal sealed class EgressPostConfigure :
+        IPostConfigureOptions<CollectionRuleOptions>
```

Comment:
Please combined all of these new `IPostConfigureOptions<CollectionRuleOptions>` implementations into one class since they are all doing the same type of work on the same options type with the same default information. Maybe call it `DefaultCollectionRulePostConfigureOptions`. Trying to avoid calling it `CollectionRuleDefaultsPostConfigureOptions` since that implies it is an `IPostConfigureOptions<CollectionRuleDefaultsOptions>`.